robotino_bringup_sim/scripts/spawn_object.py

```diff
@@ -121,30 +121,51 @@
 
 		try:
 			file_localition = roslib.packages.get_pkg_dir('cob_gazebo_objects') + '/objects/' + model + '.' + model_type
-		except:
-			print "File not found: cob_gazebo_objects" + "/objects/" + model + "." + model_type
-			continue
-
-		# call gazebo service to spawn model (see http://ros.org/wiki/gazebo)
-		if model_type == "urdf":
-			srv_spawn_model = rospy.ServiceProxy('/gazebo/spawn_urdf_model', SpawnModel)
-			file_xml = open(file_localition)
-			xml_string=file_xml.read()
-
-		elif model_type == "urdf.xacro":
-			p = os.popen("rosrun xacro xacro.py " + file_localition)
-			xml_string = p.read()
-			p.close()
-			srv_spawn_model = rospy.ServiceProxy('/gazebo/spawn_urdf_model', SpawnModel)
-
-		elif model_type == "model":
-			srv_spawn_model = rospy.ServiceProxy('/gazebo/spawn_gazebo_model', SpawnModel)
-			file_xml = open(file_localition)
-			xml_string=file_xml.read()
-		else:
-			rospy.logerr('Model type not know. model_type = ' + model_type)
-			continue
-
+		
+			# call gazebo service to spawn model (see http://ros.org/wiki/gazebo)
+			if model_type == "urdf":
+				srv_spawn_model = rospy.ServiceProxy('/gazebo/spawn_urdf_model', SpawnModel)
+				file_xml = open(file_localition)
+				xml_string=file_xml.read()
+
+			elif model_type == "urdf.xacro":
+				p = os.popen("rosrun xacro xacro.py " + file_localition)
+				xml_string = p.read()
+				p.close()
+				srv_spawn_model = rospy.ServiceProxy('/gazebo/spawn_urdf_model', SpawnModel)
+
+			elif model_type == "model":
+				srv_spawn_model = rospy.ServiceProxy('/gazebo/spawn_gazebo_model', SpawnModel)
+				file_xml = open(file_localition)
+				xml_string=file_xml.read()
+			else:
+				rospy.logerr('Model type not know. model_type = ' + model_type)
+				continue
+		except: 
+			try:
+				file_localition = roslib.packages.get_pkg_dir('robotino_gazebo_worlds') + '/objects/' + model + '.' + model_type
```

Comment:
maybe. i didnt test it much. exactly which rest objects you mean? 
 for me that  was just temporary solution which works for your objects and my custom made  simple_blue_box (i can spawn all of them separately and together